web/main/templates/includes/page_buttons.html

```diff
@@ -0,0 +1,19 @@
+<nav aria-label="Previous Next Page">
+{% load call_method %}
```

Comment:
Are you using `call_method`, or is this left over from copying/pasting?

Reply:
leftover! removed!😁